container_files/public_html/doc/builtin/sql/ValueExpression.md

```diff
@@ -595,11 +595,16 @@ The following useful non-standard aggregation functions are also supported:
   column name and value given.  This can be used with a group by clause to
   transform a dense dataset of (actor,action,value) records into a sparse
   dataset with one sparse row per actor, for example to create one-hot feature vectors or term-document or cooccurrence matrices.
-- `string_agg(expr, separator)` will coerce the value of `expr` and that of
-  `separator` to a string, and produce a single string with the concatenation
-  of `expr` separated by `separators` at internal boundaries.  For example,
-  if `expr` is `"one"`, `"two"` and `"three"` in the group, and separator is
-  `', '` the output will be `"one, two, three"`.
+- `string_agg(expr, separator [, sortField])` will coerce the value of `expr`
+   and that of `separator` to a string, sort them by the `sortField`
```

Comment:
Here _sort them_ refers to the values of `expr` which is not clear in the context.